### PR TITLE
Get filters to display from url path

### PR DESCRIPTION
**Pull request by MicheleK17**
Ticket: #260
Change-Id: Icef087c8a87bd29f3251099de4c54144ff90d3d0

### CODE REVIEW

**Review by GerberI — APPROVED** [review]
Commit: Rename QueryPageServices => Services
thx

**Review by GerberI — APPROVED** [review]
Commit: Displ. default filter, if no filter in the path

File: src/app/Main/Body/QueryPage/Services/PathServices/getFilterFromPath.service.ts [hunk 1]
```diff
@@ -9,14 +10,18 @@ import { FilterInterface, FilterType } from "../../../../../Shared/Model/Filter.
...
+): [FilterInterface, string[]] {
     const searchParams = new URLSearchParams(path);
     const filterFromPath: FilterInterface = {};
+    const displFilterFromPath: string[] = []
 
     filterKeys.forEach((filterKey) => {
         const filterValues: string[] = searchParams.getAll(filterKey);
         filterFromPath[filterKey] = filterValues;
+        if (!_.isEmpty(filterValues)) {
+            displFilterFromPath.push(filterKey)
```
[the remark this answers is not in the document]
Comment (MicheleK17) [reply src/app/Main/Body/QueryPage/Services/PathServices/getFilterFromPath.service.ts:22]: In the URL, there is no difference between a selected filter and a displayed filter. When you reload the page, there is no reason to display filters that are not also used.

File: src/app/Main/Body/QueryPage/Services/PathServices/getFilterFromPath.service.ts [hunk 2]
```diff
@@ -9,14 +10,18 @@ import { FilterInterface, FilterType } from "../../../../../Shared/Model/Filter.
 export function getFilterFromPath(
     path: string,
     filterKeys: FilterType[]
-): FilterInterface {
+): [FilterInterface, string[]] {
```
[the remark this answers is not in the document]
Comment (MicheleK17) [reply src/app/Main/Body/QueryPage/Services/PathServices/getFilterFromPath.service.ts]: Done.

**Review by GerberI — APPROVED** [review]
Commit: Change return array to object
thx

File: src/app/Main/Body/QueryPage/Services/PathServices/getFilterFromPath.service.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (GerberI) [reply src/app/Main/Body/QueryPage/Services/PathServices/getFilterFromPath.service.ts]: When there are the same, why is there a difference between selected and displayed filters here? Does a selectedFilter with an empty value array has any meaning in the app?
Add what is with the default displayed filters, are they removed when the page is refreshed?

[the remark this answers is not in the document]
Comment (GerberI) [reply src/app/Main/Body/QueryPage/Services/PathServices/getFilterFromPath.service.ts]: => ticket

**Comment by GerberI** [comment]
ready to commit

**Pull request merged**